Corrected formatting (#73)
magento/knowledge-base: src/troubleshooting/general/magento-2.4.0-checkout-error-selecting-local-payments.md

labels: 2.4.0,Braintree,Magento Commerce,Magento Commerce Cloud,billing address,checkout,known issues,payment method
---

> This article talks about a solution for known issue in Magento during checkout where an error message appears when selecting a local payment method for some countries. This occurs for the countries Belgium, Italy, Netherlands, Poland, and Spain. 

Reviewer: ```suggestion
This article talks about a solution for known issue in Adobe Commerce during checkout where an error message appears when selecting a local payment method for some countries. This occurs for the countries: Belgium, Italy, Netherlands, Poland, and Spain. 
```

Author: Reject : The problem here is that "Adobe Commerce" is the licensed version. The free version is "Magento Open Source" - making the change above implies that it is only for the version. In this case Magento is generic.

Colon not appropriate here.

Reviewer: Unfortunately, in the Support KB (you are welcome to verifying this with Sasha/Stephanie/Vicky), we do not mention Magento Open Source unless it is for a specific issue with Magento Open Source. There are several reasons why it isn't specifically mentioned in every article, including the implication that mentioning Magento Open Source in all articles implies that it is supported by Support engineers-- and Magento Open Source is not supported by Support engineers -- plus, the rebrand is to change the "Magento" generic term to "Adobe Commerce".

The colon is proper English grammar preceding a list. https://grammar.yourdictionary.com/punctuation/grammar-colon-usage.html (among other places) -- Reference Rule 1.

> The error message, " *There are currently no available payment methods. Please update your Billing Address.* " will appear, but the local payment methods will still appear and function correctly. A permanent fix will be available in Magento 2.4.1. 

Reviewer: ```suggestion
The error message, " *There are currently no available payment methods. Please update your Billing Address.* " will appear, but the local payment methods will still appear and function correctly. A permanent fix will be available in Adobe Commerce 2.4.1. 
```

Author: Reject : See above

## Issue

> ### Preconditions

Reviewer: ```suggestion
<ins>Preconditions</ins>:
```

Author: Reject : <ins> is not used in Markdown. It appears nowhere in this repo or even DevDocs

Reviewer: Actually, because of our unique setup, and style guide/format, you -must- use the "ins" html tag in order for the underline function to be shown in ZenDesk, which is where this article is published.

All Support KB articles are published in ZenDesk. This is different than what DevDocs and MerchDocs use.

I am fully aware it is not Markdown-approved, but it is the -only way- the end result can be underlined in ZD. I welcome suggestions, and you can iterate underline ways if you want to try to find a Markdown way, but no one on our end has found another reliable way to underline to fit our style guide/format.

Author: Okay thats perfectly acceptable! Also I have discussed naming nomenclature on the Slack Channel and am now happy with the decision. I just wanted to make sure it was correct.

* Magento 2.4.0 is installed.
* Create one product and one category.
* Configure [Braintree Payment Method](https://devdocs.magento.com/guides/v2.4/graphql/payment-methods/braintree.html) .

> ### Steps to reproduce

Reviewer: ```suggestion
<ins>Steps to reproduce</ins>:
```

Author: Reject : See above

Reviewer: See my above.

1. Navigate to the storefront.
1. Select items to add to the cart.
1. Proceed to checkout.
1. Fill out the address form a with valid address.
1. Proceed to the Review & Payments page.

> ### Expected result

Reviewer: ```suggestion
<ins>Expected result</ins>:
```

Author: Reject : See above

The local payment methods should be displayed normally, without an error message.

> ### Actual result

Reviewer: ```suggestion
<ins>Actual result</ins>:
```

Author: Reject : See above

The error message, " *There are currently no available payment methods. Please update your Billing Address.* " appears, but the local payment methods will still display and function correctly.

## Solution

> The solution is to ignore the displayed error message, and continue with payment as normal, as all local payment methods will function normally. A fix will be available in Magento version 2.4.1, which is scheduled for release in Q4 1. 

Reviewer: ```suggestion
The solution is to ignore the displayed error message, and continue with payment as normal, as all local payment methods will function normally. The fix was available starting with Adobe Commerce version 2.4.1. 
```

Author: Will reword, but again the use of "Adobe Commerce" implies it is for the licensed version only.

## Related reading

> * [Magento 2.4.0 known issue: raw message data display on storefront](https://support.magento.com/hc/en-us/articles/360045804332)

Reviewer: ```suggestion
* [Adobe Commerce 2.4.0 known issue: raw message data display on storefront](https://support.magento.com/hc/en-us/articles/360045804332)
```

Author: Reject : Same as above

> * [Magento 2.4.0 known issue: Export Tax Rates does not work](https://support.magento.com/hc/en-us/articles/360045850032)

Reviewer: ```suggestion
* [Adobe Commerce 2.4.0 known issue: Export Tax Rates does not work](https://support.magento.com/hc/en-us/articles/360045850032)
```

Author: Reject : Same as above
